src/routes/services.test.js

```diff
@@ -0,0 +1,353 @@
+import { expect } from 'chai';
+import config from 'config';
+import nock from 'nock';
+import request from 'supertest';
+
+import app, { basePath } from '../index.js';
+
+export const COLLECTIONS_RESULT = {
+  'Collection 1': {
+    id: 'collection-1',
+    endpoint: 'http://collection-1.org/api/v1',
+  },
+  'Collection 2': {
+    id: 'collection-2',
+    endpoint: 'http://collection-2.org/api/v1',
+  },
+};
+
+const COLLECTION_1_SERVICES_RESULT = [
+  {
+    id: 'service-1',
+    name: 'Service 1',
+    terms: [
+      { type: 'Privacy Policy' },
+    ],
+  },
+  {
+    id: 'service-2',
+    name: 'Service 2',
```

Comment:
Increase variability to use different test classes in order to improve tests relevance.
```suggestion
    id: 'Service with Chinese characters',
    name: '抖音短视频',
```